lib/util/call-taker.js

```diff
@@ -269,3 +270,64 @@ function createItinerary (itinData, tripPlan) {
   })
   return itin
 }
+
+/**
+ * LZW-compress a string
+ *
+ * LZW functions adapted from jsolait library (LGPL)
+ * via http://stackoverflow.com/questions/294297/javascript-implementation-of-gzip
+ */
+export function lzwEncode (s) {
```

Comment:
Is this used anywhere? I'm guessing it will be at some point?

Reply:
Possibly in @evansiroky's PR. I'll leave it there.

Reply:
Yeah, I'll need this.